js/main.js

```diff
@@ -0,0 +1,108 @@
+var cdYears = document.getElementById("cd-years");
```

Comment:
Could you fix the indentations of this file? Should be moved left

Reply:
Should be resolved now!